config/lib/app.js

```diff
@@ -47,6 +47,7 @@ module.exports.start = function start(callback) {
       console.log(chalk.green('App version:     ' + config.meanjs.version));
       if (config.meanjs['meanjs-version'])
         console.log(chalk.green('MEAN.JS version: ' + config.meanjs['meanjs-version']));
+      console.log(chalk.green('Node version:    ' + process.version.substr(1)));
```

Comment:
@hyperreality please ping me if this PR goes in and I'll remove [similar line](https://github.com/meanjs/mean/pull/1655/commits/875f2c104501f9c716fe76f3cad644d140e6f226#diff-e4a0ef5ca61cb235bb9ed42cac9853ffR47) from my PR.